tox.ini

```diff
@@ -1,41 +1,33 @@
 [tox]
 downloadcache = {toxworkdir}/_download/
-envlist = py26-1.4.X,py26-1.3.X,py26-1.2.X,docs
+envlist =
+    {py26,py27}-{django1.6.X},
+    {py27}-{django1.7.X},
+    docs
 
 [testenv]
-commands = django-admin.py test decisiontree
+basepython =
+    py26: python2.6
+    py27: python2.7
+
+commands =
+    coverage run {envbindir}/django-admin.py test decisiontree
+    flake8
+    coverage report -m
```

Comment:
Of course you're right! I'm fine either way... either keeping it like you have or just running once on the latest combination.